package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "ft-next-syndication-api",
   "description": "Next Syndication API",
-  "version": "0.38.0",
+  "version": "0.39.0",
```

nitpick: Should be a patch release as it is a bug fix
https://semver.org/#spec-item-6

```suggestion
  "version": "0.38.1",
```